examples/sbt-scala/blackbox-test/steps/service_steps.rb

```diff
@@ -0,0 +1,13 @@
+step ":host is listening on :port" do |host, port|
+  eventually(timeout: 120) {
+    `nc -z #{host} #{port}`
+    expect($?).to be_success
+  }
+end
+
+step "the homepage should contain :content" do |content|
+  visit("/")
```

Comment:
I think this needs to change to `visit("/hello")`.

When I run the current code locally the `Home page` scenario fails with:

```
Failures:

  1) Docker image should be serving content Home page Given prodserver is listening on 8080 -> Then the homepage should contain "hello"
     Failure/Error: Then the homepage should contain "hello"
       expected to find text "hello" in "The requested resource could not be found."
```